pkgs/games/quake3/ioquake/default.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation {
   meta = with stdenv.lib; {
     homepage = "https://ioquake3.org/";
     description = "First person shooter engine based on the Quake 3: Arena and Quake 3: Team Arena";
-    license = licenses.gpl2;
+    license = licenses.gpl2Only;
```

Comment:
Wasn't sure which one to select, but this one seems the most applicable https://github.com/ioquake/ioq3/blob/05180e32dcfb9a4552e1b9652b56127248a9950c/COPYING.txt